lib/core/network/models/user_profile_data.freezed.dart

```diff
@@ -64,6 +70,10 @@ class _$UserProfileDataCopyWithImpl<$Res, $Val extends UserProfileData>
           ? _value.accountName
           : accountName // ignore: cast_nullable_to_non_nullable
               as String,
+      network: null == network
+          ? _value.network
+          : network // ignore: cast_nullable_to_non_nullable
```

Comment:
maybe we can default existing profiles to Network.telos - compatibility for old accounts?